isofit/utils/template_construction.py

```diff
@@ -550,123 +440,6 @@ def get_grid(minval: float, maxval: float, spacing: float, min_spacing: float):
         return grid
 
 
-def get_angular_grid(
```

Comment:
Did we have this function twice in `template_construction.py`?

Reply:
We did - once as a class function and once as a standalone.